base/libgit2/blob.jl

```diff
@@ -8,10 +8,23 @@ function Base.length(blob::GitBlob)
     return ccall((:git_blob_rawsize, :libgit2), Int64, (Ptr{Void},), blob.ptr)
 end
 
+function isbinary(blob::GitBlob)
+    bin_flag = ccall((:git_blob_isbinary, :libgit2), Int64, (Ptr{Void},), blob.ptr)
```

Comment:
I think this should actually be `git_blob_is_binary`. See [the libgit2 API docs](https://libgit2.github.com/libgit2/#HEAD/group/blob/git_blob_is_binary).